### DISCUSSION TITLE
ERROR: not enough PSBT output found to put all required state (can't add assignment type 4000 for unspecified-velocity state).

### DISCUSSION THREAD

**Discussion by cymqqqq** [RGB-WG/rgb#98] [Q&A] [answered]
https://github.com/RGB-Tools/rgb-sandbox/issues/4
Does anyone know about this error? ERROR: not enough PSBT output found to put all required state (can't add assignment type 4000 for unspecified-velocity state).

**Comment by dr-orlovsky** [comment, marked as answer]
I think this should go into RGB-Tools and not here. This error means that the wallet did not produce enough outputs to store RGB state
